cns/networkcontainers/networkcontainers_linux.go

```diff
@@ -13,10 +35,139 @@ func setWeakHostOnInterface(ipAddress string) error {
 	return nil
 }
 
-func createOrUpdateWithOperation(createNetworkContainerRequest cns.CreateNetworkContainerRequest, operation string) error {
+func updateInterface(createNetworkContainerRequest cns.CreateNetworkContainerRequest, netpluginConfig *NetPluginConfiguration) error {
+	log.Printf("[Azure CNS] update interface operation called.")
+
+	// Currently update via CNI is only supported for ACI type
+	if createNetworkContainerRequest.NetworkContainerType == cns.WebApps {
+		log.Printf("[Azure CNS] operation not supported for WebApps Orchestrator.")
+		return nil
+	}
+
+	if _, err := os.Stat(netpluginConfig.path); err != nil {
+		if os.IsNotExist(err) {
+			log.Printf("[Azure CNS] Unable to find plugin " + netpluginConfig.path)
+			return errors.New("[Azure CNS] Unable to find " + netpluginConfig.path + ", cannot continue.")
+		}
+	}
+
+	var podInfo cns.KubernetesPodInfo
+	err := json.Unmarshal(createNetworkContainerRequest.OrchestratorContext, &podInfo)
+	if err != nil {
+		log.Printf("[Azure CNS] Unmarshalling %s failed with error %v", createNetworkContainerRequest.NetworkContainerType, err)
+		return err
+	}
+
+	log.Printf("[Azure CNS] Pod info %v", podInfo)
+
+	rt, err := buildCNIRuntimeConf(podInfo.PodName, podInfo.PodNamespace, "", "", createNetworkContainerRequest.NetworkContainerid)
+	if err != nil {
+		log.Printf("[Azure CNS] Failed to build runtime configuration with error %v", err)
+		return err
+	}
+
+	log.Printf("[Azure CNS] run time conf info %v", rt)
+
+	netConfig, err := getNetworkConfig(netpluginConfig.networkConfigPath)
+	if err != nil {
+		log.Printf("[Azure CNS] Failed to build network configuration with error %v", err)
+		return err
+	}
+
+	log.Printf("[Azure CNS] network configuration info %v", string(netConfig))
```

Comment:
since you've been adding spaces around log.Printf above, let's add space around this one too